pyresample/test/test_sgeom/__init__.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2021 Pyresample developers
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Test for spherical geometries."""
```

Comment:
Should this module maybe be called `test_future_spherical.py`? Or maybe rename the old one to `test_legacy_spherical.py` and name this `test_spherical.py`?

Reply:
I agree that the directory should be defined as `test_spherical` .
I have 5 ongoing PR that use now `test_sgeom` directory.
If for you is ok, I will rename it after the merging of these PRs ...

Reply:
Sounds good to me. Thanks.